### DISCUSSION TITLE
Delete my own discussions

### DISCUSSION THREAD

**Discussion by fnoquiq** [community/community#3435] [Discussions] [answered]
We cant delete discussions post

**Comment by munderseth** [comment]
A discussion post can be deleted. Refer to the right side of the page.

![image](https://user-images.githubusercontent.com/1229367/109337789-39415d80-781a-11eb-9528-fa0e8e009551.png)

**Comment by fnoquiq** [comment]
![1](https://user-images.githubusercontent.com/11557379/109361577-f4441800-7867-11eb-9f5e-328afd75d03f.PNG)

**Comment by munderseth** [comment, marked as answer]
What is your `Access level` for the specific repo? In the example I provided I am an `Admin`.

**Reply by fnoquiq** [reply]
I have no access. However, by default should I be allowed to delete my own posts?

**Reply by Zerotask** [reply]
I agree with @fnoquiq the owner / author should always have the rights to delete his discussion.

**Reply by ghost** [reply]
Well, as other might be involved, deleting the discussion might not be an option (loss of historical context) but the ability to "dis-associate" the authors username from the post would be helpful.

**Reply by Levi-Lesches** [reply]
@Zerotask and @fnoquiq, This isn't possible for issues, I don't see why discussions would be any different. Both issues and discussions containn conversation, and often historical context. Deleting that would be robbing the repo of its own history.
